### ISSUE TITLE
A11y: Labeling of the switches for removing a filter could be improved

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7108]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

Labeling of the switches for removing a filter is not easily understanable for people having visual or cognitive impairments. There is already an `aria-label`, describing the function of the X-button for people using screen readers.

*_Expected behaviour:_*

The same information that is included in the existing `aria-label` should be visually provided to users hovering over the element as a tooltip or using the `title` attribute.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. navigate to a page containing product filters
2. set a filter
3. hover over the X-button to remove a filter.

![Image](https://github.com/user-attachments/assets/2f8335de-ba87-4376-a072-dc4e15b9af00)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40864**



[created from NEXT-40864, comment 541930]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **Low**

 Thank you for your report. After careful consideration, we've concluded that this issue does not require immediate attention or isn't something we can address right now. We prioritize openness and clear communication with our community, so we want to be transparent. We may not resolve all reported issues, and this one might remain unresolved. To stay effective, we periodically clean our backlog, and this issue could be closed in the future.

 For now, we'll keep this ticket open to invite further feedback from the community in case we've underestimated its impact and to allow us to revisit it later.



[created from NEXT-40864, comment 541979]
